client/X11/xf_client.c

```diff
@@ -71,6 +71,13 @@
 #include <freerdp/codec/color.h>
 #include <freerdp/codec/bitmap.h>
 
+/*
+ * X11/Xlib.h defines Status, but winpr/rpc.h, which is eventually included by
+ * mcs.h, declares a few functions that have a parameter named Status.
+ */
+#undef Status
+#include <libfreerdp/core/mcs.h>
```

Comment:
Sorry, no includes from the `libfreerdp` folder. They are not part of the exposed API

Reply:
Okay.  This commit does add the reason to the exposed API though.  Does that mean that the `MCS_Reason` enum should be mvoed into one of the headers in include/freerdp?  Maybe constants.h where a few other constants are defined?  Or maybe freerdp.h where the `rdp_context` structure that contains the reason field is defined?

Reply:
@r-barnett good question, the name is bound to the `mcs` module. Might be a good idea to move and rename that to `freerdp.h`